src/main/scala/piconot/RightHand.scala

```diff
@@ -0,0 +1,48 @@
+package piconot
+
+/**
+ * @author aputman
+ */
+object RightHand extends PracticalBot {
+  // This is actually the LEFT hand rule :^)
+
+  // Ideally, the maze statement would go somewhere else?
+  // We might want a way for users to add keywords into their programs
+  // The program relies on the user using reasonable state names for 
+  // program readability
+  picobot("maze") (
+    // future: removing parens for the state names
```

Comment:
As I've been dropping in others' assignments, here's a link for removing parens.
http://stackoverflow.com/questions/1181533/what-are-the-precise-rules-for-when-you-can-omit-parenthesis-dots-braces-f